link-global.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
```

Comment:
have you looked at the `./link-all.sh` script? for us it's usually been sufficient for local development.

Reply:
When I work on project using `cdk` I would like to use development versions to generate infrastructure and to test how my changes actually works. 

I can use in my project `npm link @aws-cdk/XYZ` so it will create all required symbolic links there (sometimes I need to calla agin `link-all`)

Reply:
Actually I'm linking are CDK packages in my project with
```
for f in node_modules/@aws-cdk/*; do npm link "@aws-cdk/$(basename $f)"; done
```